pom.xml

```diff
@@ -44,6 +44,7 @@
     <properties>
         <jenkins.version>2.60.3</jenkins.version>
         <java.level>8</java.level>
+        <jcasc.version>1.27</jcasc.version>
```

Comment:
Why not latest version?

Reply:
It rolls fast.

```suggestion
        <jcasc.version>1.29</jcasc.version>
```

Reply:
I knew that 1.27 had core 2.60.x and didn't bother to investigate if newer versions existed and had the same core dependency :)